am/current/management-api/3.20/swagger.json

```diff
@@ -0,0 +1 @@
nticationDeviceNotifier":{"type":"object","required":["configuration","name","type"],"properties":{"id":{"type":"string"},"type":{"type":"string"},"name":{"type":"string"},"configuration":{"type":"string"}}},"ThemeEntity":{"type":"object","properties":{"id":{"type":"string"},"referenceId":{"type":"string"},"referenceType":{"type":"string","enum":["PLATFORM","DOMAIN","APPLICATION","ORGANIZATION","ENVIRONMENT"]},"logoUrl":{"type":"string"},"logoWidth":{"type":"integer","format":"int32"},"faviconUrl":{"type":"string"},"primaryButtonColorHex":{"type":"string"},"secondaryButtonColorHex":{"type":"string"},"primaryTextColorHex":{"type":"string"},"secondaryTextColorHex":{"type":"string"},"css":{"type":"string"},"createdAt":{"type":"integer","format":"int64"},"updatedAt":{"type":"integer","format":"int64"}}},"NewTheme":{"type":"object","properties":{"logoUrl":{"type":"string"},"logoWidth":{"type":"integer","format":"int32"},"faviconUrl":{"type":"string"},"primaryButtonColorHex":{"type":"string"},"secondaryButtonColorHex":{"type":"string"},"primaryTextColorHex":{"type":"string"},"secondaryTextColorHex":{"type":"string"},"css":{"type":"string"}}},"Audit":{"type":"object","properties":{"id":{"type":"string"},"transactionId":{"type":"string"},"type":{"type":"string"},"referenceType":{"type":"string","enum":["PLATFORM","DOMAIN","APPLICATION","ORGANIZATION","ENVIRONMENT"]},"referenceId":{"type":"string"},"accessPoint":{"$ref":"#/definitions/AuditAccessPoint"},"actor":{"$ref":"#/definitions/AuditEntity"},"target":{"$ref":"#/definitions/AuditEntity"},"outcome":{"$ref":"#/definitions/AuditOutcome"}}},"AuditAccessPoint":{"type":"object","properties":{"id":{"type":"string"},"alternativeId":{"type":"string"},"displayName":{"type":"string"},"ipAddress":{"type":"string"},"userAgent":{"type":"string"}}},"AuditEntity":{"type":"object","properties":{"id":{"type":"string"},"alternativeId":{"type":"string"},"type":{"type":"string"},"displayName":{"type":"string"},"referenceType":{"type":"string","enum":["PLATFORM","DOMAIN","APPLICATION","ORGANIZATION","ENVIRONMENT"]},"referenceId":{"type":"string"},"attributes":{"type":"object","additionalProperties":{"type":"object"}}}},"AuditOutcome":{"type":"object","properties":{"status":{"type":"string"},"message":{"type":"string"}}},"ScopeApprovalEntity":{"type":"object","properties":{"id":{"type":"string"},"transactionId":{"type":"string"},"userId":{"type":"string"},"clientId":{"type":"string"},"domain":{"type":"string"},"scope":{"type":"string"},"status":{"type":"string","enum":["APPROVED","DENIED"]},"expiresAt":{"type":"integer","format":"int64"},"createdAt":{"type":"integer","format":"int64"},"updatedAt":{"type":"integer","format":"int64"},"clientEntity":{"$ref":"#/definitions/ApplicationEntity"},"scopeEntity":{"$ref":"#/definitions/ScopeEntity"}}},"ScopeEntity":{"type":"object","properties":{"id":{"type":"string"},"key":{"type":"string"},"name":{"type":"string"},"description":{"type":"string"}}},"Credential":{"type":"object","pro
```

Comment:
Should there be some whitespace in this file?

Reply:
this is a auto generated file and historically we haven't added whitespaces.  We can ignore it.